python/mxboard/utils.py

```diff
@@ -153,7 +153,7 @@ def norm_range_helper(t, val_range):
     # make the batch of images into a grid
     nmaps = tensor.shape[0]
     xmaps = min(nrow, nmaps)
-    ymaps = int(np.ceil(float(nmaps) / xmaps))
+    ymaps = xmaps
```

Comment:
This function is also called by `add_image` in addition to `add_embedding`, and we don't want to change the behavior of `add_image`. Could you add a parameter called `square_image=False` to `make_image_grid` and its callers, and only use `square_image=True` when called from `add_embedding`?

Reply:
Hi reminisce,

Many thanks for your notification. I'll edit it according to your suggestion and get it back to you soon!